### ISSUE TITLE
Error occurs from pqos -I

### ISSUE THREAD

**Issue by poopbear** [intel/intel-cmt-cat#78]
**pqos -I**
```
root@csl-cat:/home/kyuyeonpooh# pqos -I
NOTE:  Mixed use of MSR and kernel interfaces to manage
       CAT or CMT & MBM may lead to unexpected behavior.
WARN: As of Kernel 4.10, Intel(R) RDT perf results per core are found to be incorrect.
ERROR: Failed to start all selected OS monitoring events
Monitoring start error on core(s) 0, status 1
```

**pqos -I -p**
```
root@csl-cat:/home/kyuyeonpooh# pqos -I -p
NOTE:  Mixed use of MSR and kernel interfaces to manage
       CAT or CMT & MBM may lead to unexpected behavior.
Monitoring top-pids enabled
ERROR: Failed to start all selected OS monitoring events
PID 2517 monitoring start error,status 1
```

**pqos -I -s**: I guess this works pretty fine
```
root@csl-cat:/home/kyuyeonpooh# pqos -I -s
NOTE:  Mixed use of MSR and kernel interfaces to manage
       CAT or CMT & MBM may lead to unexpected behavior.
L3CA/MBA COS definitions for Socket 0:
    L3CA COS0 => MASK 0x0
    L3CA COS1 => MASK 0x0
    L3CA COS2 => MASK 0x0
    L3CA COS3 => MASK 0x0
    L3CA COS4 => MASK 0x0
    L3CA COS5 => MASK 0x0
    L3CA COS6 => MASK 0x0
    L3CA COS7 => MASK 0x0
    MBA COS0 => 100% available
    MBA COS1 => 100% available
    MBA COS2 => 100% available
    MBA COS3 => 100% available
    MBA COS4 => 100% available
    MBA COS5 => 100% available
    MBA COS6 => 100% available
    MBA COS7 => 100% available
Core information for socket 0:
    Core 0, L2ID 0, L3ID 0 => COS0
    Core 1, L2ID 1, L3ID 0 => COS0
    Core 2, L2ID 2, L3ID 0 => COS0
    Core 3, L2ID 3, L3ID 0 => COS0
    Core 4, L2ID 4, L3ID 0 => COS0
    Core 5, L2ID 5, L3ID 0 => COS0
    Core 6, L2ID 6, L3ID 0 => COS0
    Core 7, L2ID 7, L3ID 0 => COS0
    Core 8, L2ID 0, L3ID 0 => COS0
    Core 9, L2ID 1, L3ID 0 => COS0
    Core 10, L2ID 2, L3ID 0 => COS0
    Core 11, L2ID 3, L3ID 0 => COS0
    Core 12, L2ID 4, L3ID 0 => COS0
    Core 13, L2ID 5, L3ID 0 => COS0
    Core 14, L2ID 6, L3ID 0 => COS0
    Core 15, L2ID 7, L3ID 0 => COS0
PID association information:
    COS1 => (none)
    COS2 => (none)
    COS3 => (none)
    COS4 => (none)
    COS5 => (none)
    COS6 => (none)
    COS7 => (none)
```
**pqos -I -R**
```
root@csl-cat:/home/kyuyeonpooh# pqos -I -R
NOTE:  Mixed use of MSR and kernel interfaces to manage
       CAT or CMT & MBM may lead to unexpected behavior.
ERROR: Invalid file arguments!
ERROR: Error on schemata write for resctrl group 1
Allocation reset failed!
```

**CPU: Intel Xeon Silver 4110 CPU @ 2.10GHz**
**Kernel: 4.13.0-45-generic**
**OS: Ubuntu 16.04 amd64**

When do these issues occur?
There have been no problem using MSR interfaces.
After resetting all the interfaces and I started to look for OS interface for process-degree manipulation, however, several unknown errors occurred.

**Comment by mdcornu** [comment]
Hi poopbear,

Please see the response posted for a similar issue #73.
I believe you are encountering the same problem.

For instructions on how to enable these features in the kernel, please see our wiki:
https://github.com/intel/intel-cmt-cat/wiki#enabling-intelr-rdt-support-in-the-linux-kernel

Regards,
Marcel

**Comment by poopbear** [comment]
Dear Marcel,

Thank you for your fast reply and your advice.

I'll give it a try!

Sincerely, Kyuyeon.

**Comment by poopbear** [comment]
This issue is solved, which is related to issue #73.
By adding parameters in kernel command line, this issue is solved.
To add boot parameters in kernel command you can refer here[https://askubuntu.com/questions/19486/how-do-i-add-a-kernel-boot-parameter]
